src/docs/admin/AdminTutorials.adoc

```diff
@@ -2,42 +2,85 @@
 include::../common-settings.adoc[]
 :toc-title: Tutorials for Administrators
 
-== Step by Step Tutorials
+== Run As Me
+This is a step by step tutorial, which was tested on Ubuntu 14.04 and Microsoft Windows 7.
 
-=== Run As Me
-This is a step by step tutorial, which was designed and tested for Ubuntu 14.04.
-
-ProActive Workflows & Scheduling allows advanced user management with the “run as me” feature. Run as me allows
+ProActive Workflows & Scheduling allows advanced user management with the *Run As Me* feature. Run as me allows
 to execute a task under the current ProActive user, if an equally named system user exists on the executing machine.
 
-*Following steps configure the Node with a sample user called Gord:*
+=== Node configuration
+
+The following steps configure the Node with a sample user called Gord.
 
-. Create a group called proactive `groupadd proactive’
-. Create a user running the ProActive Node, set it's primary group to proactive
- ‘useradd -g proactive -G sudo -m proactive’
-. Set the password of proactive user ‘passwd proactive’
+==== Linux
+
+. Create a group called activeeon: _groupadd activeeon_
+. Create a user running the ProActive Node, set it's primary group to proactive: _useradd -g activeeon -G sudo -m proactive_
+. Set the password of proactive user: _passwd proactive_
 . Download and unzip ProActive inside the /opt/ directory (/opt/ gives execution right to everyone)
-. Change ownership of the ProActive folder with ‘sudo chown -R proactive:proactive ProActiveNode’
-. Add a run as me user, which is part of the proactive group ‘useradd -g proactive -m Gord’
-. Create an ssh key for the run as me user (Gord) ‘ssh-keygen -t rsa’
-. Go into the /home/Gord/.ssh directory and copy Gord’s public key to the authorized keys ‘cat id_rsa.pub >> authorized_keys’
-. Log into the proactive account
-. Start the ProActive Node with the property ‘-Dpas.launcher.forkas.method=key’
+. Change ownership of the ProActive folder with _sudo chown -R proactive:activeeon ProActive_
+. Add a *Run As Me* user, which needs to be part of the activeeon group _useradd -g activeeon -m Gord_
+. Create an ssh key for the *Run As Me* user (Gord) _ssh-keygen -t rsa_
+. Go into the /home/Gord/.ssh directory and copy Gord’s public key to the authorized keys _cat id_rsa.pub >> authorized_keys_
+. Log into the activeeon account
+. Start the ProActive Node with the property `-Dpas.launcher.forkas.method=key`
+
+NOTE: The proactive user got a password set with _passwd proactive_ to execute the sudo command. Whereas the run as me user (Gord) got a SSH key set. If you wish you can assign a password to Gord _passwd Gord_.
+
+
+==== Windows
+
+. Create an account called proactive:
```

Comment:
In the generated HTML there are two items labeled _1_. One for _Create an account called proactive:_ and another for _Create an account Gord that will be used as the *Run As Me* account_.